src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -708,6 +709,11 @@
             .put("west", (BigMushroomType) (Object) BlockHugeMushroom.EnumType.WEST)
             .build();
     
+    private final Map<String, SkeletonType> skeletonTypeMappings = new ImmutableMap.Builder<String, SkeletonType>()
```

Comment:
Please update your registry as the entire registry was changed. 

In short: Make a `SkeletonTypeRegistryModule` make the map, add the `RegisterCatalog` annotation, and add in `CommonModuleRegistry` `registry.registerModule(SkeletonType.class, new SkeletonTypeRegistryModule);`